tests/integration/components/frost-link-test.js

```diff
@@ -9,18 +18,200 @@ describeComponent(
     integration: true
   },
   function () {
-    it('renders', function () {
-      // Set any properties with this.set('myProperty', 'value');
-      // Handle any actions with this.on('myAction', function(val) { ... });
-      // Template block usage:
-      // this.render(hbs`
-      //   {{#frost-link}}
-      //     template content
-      //   {{/frost-link}}
-      // `);
-
-      this.render(hbs`{{frost-link 'test'}}`)
-      expect(this.$()).to.have.length(1)
+    it('renders default values', function () {
+      this.render(hbs`
+        {{#frost-link 'test'}}
+          Yielded content
+        {{/frost-link}}
+      `)
+
+      expect(
+        this.$('.frost-link'),
+        'has the correct class'
+      ).to.have.length(1)
+
+      expect(
+        this.$('.frost-link').text().trim(),
+        'Yields content'
+      ).to.eql('Yielded content')
+    })
+
+    describe('Priority property', function () {
+      it('has primary class set', function () {
+        this.render(hbs`
+          {{#frost-link
+            'test'
```

Comment:
Just a preference, but I like to see the positional params inline with the component (I kind of see them as function parameters)

e.g. 

```
{{#frost-link 'test'
  priority='primary'
}}
  Yielded content
{{/frost-link}}
```